File: csharp/Betfair.ESAClient/Betfair.ESAClient/Protocol/ChangeType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Betfair.ESAClient.Protocol
{
    /// <summary>
    /// Common change type (as change type is local to market / order in swagger).
    /// </summary>
    public enum ChangeType
    {
        /// <summary>
        /// Update
        /// </summary>
        UPDATE,
        /// <summary>
        /// Initial subscription image
        /// </summary>
        SUB_IMAGE,
        /// <summary>
        /// Resubscription delta image
        /// </summary>
        RESUB_DELTA,
        /// <summary>
        /// Heartbeat
        /// </summary>
        HEARTBEAT,
    }
}

File: java/client/src/main/java/com/betfair/esa/client/protocol/ChangeType.java
package com.betfair.esa.client.protocol;

/**
 * Common change type (as change type is local to market / order in swagger). Created by mulveyj on
 * 07/07/2016.
 */
public enum ChangeType {
    /** Update */
    UPDATE,
    /** Initial subscription image */
    SUB_IMAGE,
    /** Resubscription delta image */
    RESUB_DELTA,
    /** Heartbeat */
    HEARTBEAT,
}
